app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1623,29 +1629,42 @@ table li .account_details {
   height: 50px;
 }
 
-.item-title a {
-  color: white;
-}
-
 .item-price td{
   font-size: 23px;
   height: 40px;
 }
 
 .item-description p{
-  height: 65px;
+  height: 60px;
   font-size: 23px;
 }
 
-.add-to-cart td{
+.add-to-cart a{
+  padding: 10px;
   border: 5px outset gray;
+  font-size: 15px;
 }
+// end non-admin items menu
 
-.add-to-cart a{
-  font-family: 'Cardinal', Arial, sans-serif;
-  color: white;
+// start admin items menu
+.admin-menu {
+  margin-bottom: 30px;
+  font-family: 'SketchGothic', Arial, sans-serif;
+  font-size: 72px;
 }
-// end non-admin items menu
+
+.item-status td {
+  font-size: 20px;
+}
+
+.add-new-item a {
+  padding: 10px;
+  border: 5px outset gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.